weaver/common/protos-go/common/events.pb.go

```diff
@@ -1,7 +1,11 @@
+// Copyright IBM Corp. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.30.0
-// 	protoc        v3.17.3
+// 	protoc        v3.12.4
```

Comment:
@outsidethecode Why the `protoc` downgrade? Unless there's a strong reason I'd advocate  for an upgrade by default instead of a downgrade. In this specific case sticking with the existing version would be the best just because the diff is already huge and sticking with the existing version would make it a little smaller and easier to review.

Reply:
@outsidethecode Please revert the `protoc` version change in every generated Go file under `protos-go`. I suspect the contents are identical to what's in main but the version number in the comments is different, reflecting the version you've installed in your system.